wgpu/src/lib.rs

```diff
@@ -2205,16 +2206,16 @@ impl Adapter {
 }
 
 impl Device {
-    /// Check for resource cleanups and mapping callbacks.
+    /// Check for resource cleanups and mapping callbacks. Will block if [`Maintain::Wait`] is passed.
     ///
     /// Return `true` if the queue is empty, or `false` if there are more queue
     /// submissions still in flight. (Note that, unless access to the [`Queue`] is
     /// coordinated somehow, this information could be out of date by the time
     /// the caller receives it. `Queue`s can be shared between threads, so
     /// other threads could submit new work at any time.)
     ///
-    /// On the web, this is a no-op. `Device`s are automatically polled.
-    pub fn poll(&self, maintain: Maintain) -> bool {
+    /// When running on WebGPU, this is a no-op. `Device`s are automatically polled.
```

Comment:
Should the text of `Maintain`'s variant documentation also be changed to match?

Reply:
Adjusted those docs.

Reply:
The return type was changed too but the doc comments were never updated. It's not obvious to a user like me how the variants of `MaintainResult` maps to `bool` pre-change.